src/Components/PostList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Post from './Post';
+
+const PostList = ({ posts }) => (
+  <div className="container">
+    {posts.map(post => (
+      <Post post={post} key={post.id} />
```

Comment:
Три слова post подряд - bad practice
```suggestion
      <Post postData={currentPost} key={post.id} />
```

Reply:
+